common/app/com/commercetools/sunrise/common/sessions/StorableResource.java

```diff
@@ -0,0 +1,24 @@
+package com.commercetools.sunrise.common.sessions;
+
+import javax.annotation.Nullable;
+
+/**
+ * Enables a particular object to be stored somewhere.
+ * The interface does not define where (e.g. session, cache) or how (e.g. the entire object, only specific parts)
+ * the object is stored, which should be specified by the implementation class.
+ * @param <T> Class of the stored object
+ */
+public interface StorableResource<T> {
```

Comment:
If it is called `StorableResource` then sounds like the object that will be stored needs to extend this interface. `ResourceStore` would fit much better.